ios/Runner/GoogleService-Info.plist

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CLIENT_ID</key>
+	<string>345803967238-hdhsdljofjn69jijg967jj0o69nkffdm.apps.googleusercontent.com</string>
+	<key>REVERSED_CLIENT_ID</key>
+	<string>com.googleusercontent.apps.345803967238-hdhsdljofjn69jijg967jj0o69nkffdm</string>
+	<key>ANDROID_CLIENT_ID</key>
+	<string>345803967238-pu417p3rj09eku6f1d4okmvn9lifv6io.apps.googleusercontent.com</string>
+	<key>API_KEY</key>
+	<string>AIzaSyAP-Q-cbRiSy_nH8sv4InhUr7IBjmxVcpM</string>
+	<key>GCM_SENDER_ID</key>
+	<string>345803967238</string>
+	<key>PLIST_VERSION</key>
+	<string>1</string>
+	<key>BUNDLE_ID</key>
+	<string>com.example.projectEurekaFlutter</string>
+	<key>PROJECT_ID</key>
+	<string>project-eureka-firebase</string>
+	<key>STORAGE_BUCKET</key>
+	<string>project-eureka-firebase.appspot.com</string>
+	<key>IS_ADS_ENABLED</key>
+	<false></false>
+	<key>IS_ANALYTICS_ENABLED</key>
+	<false></false>
+	<key>IS_APPINVITE_ENABLED</key>
+	<true></true>
+	<key>IS_GCM_ENABLED</key>
+	<true></true>
+	<key>IS_SIGNIN_ENABLED</key>
+	<true></true>
+	<key>GOOGLE_APP_ID</key>
+	<string>1:345803967238:ios:9253fabf87713585d8ac87</string>
+</dict>
+</plist>
```

Comment:
Please delete this file. Without a Mac, importing and installing this causes problems. We can add this in the future.